src/jit/CMakeLists.txt

```diff
@@ -225,6 +222,10 @@ if ((CLR_CMAKE_PLATFORM_ARCH_I386 OR CLR_CMAKE_PLATFORM_ARCH_AMD64) AND WIN32)
     # On x86, build RyuJIT/ARM32 cross-compiling altjit.
     # On amd64, build RyuJIT/ARM64 cross-compiling altjit.
     add_subdirectory(protononjit)
+
+    # Build Linux/x86-running-on-Windows altjit or
+    # Linux/AMD64-running-on-Windows altjit..
+    add_subdirectory(linuxnonjit)
```

Comment:
btw, I agree about the names not being great. We have a matrix of host arch, host platform, target arch, target platform, and JIT flavor (legacy, RyuJIT) to accommodate.